ansible_navigator/runner/api.py

```diff
@@ -173,7 +173,6 @@ def generate_run_command_args(self) -> None:
         """generate arguments required to be passed to ansible-runner"""
         if self._playbook:
             self._cmdline.append(self._playbook)
-            self._runner_args.update({"cwd": os.path.dirname(os.path.abspath(self._playbook))})
```

Comment:
Unsure if this is safe to do or not. Tests pass, and it fixes the issue in the title of this PR, but I don't know what other effects it has. /cc @cidrblock